polling/assets.go

```diff
@@ -67,9 +69,10 @@ type PegItems struct {
 	When  string
 }
 
-func (p *PegItems) Clone() PegItems {
+func (p *PegItems) Clone(randomize float64) PegItems {
 	np := new(PegItems)
-	np.Value = p.Value
+	np.Value = p.Value + p.Value*(randomize/2*rand.Float64()) - p.Value*(randomize/2*rand.Float64())
```

Comment:
I am not sure why we have randomization code in the middle production code. If this is for testing, we can find a proper way to have this code extracted and used in the test execution path only. But I don't see a test file in this PR either so I am not sure I understand.

Reply:
That can be done better.  I'll work on it.